app/Resources/views/full/home.html.twig

```diff
@@ -1,8 +1,21 @@
 {% extends "pagelayout.html.twig" %}
```

Comment:
It's because you are missing:

```yml
 system:
      default:
          pagelayout: pagelayout.html.twig
```

add `pagelayout` key below https://github.com/SylvainGuittard/ezplatform-com/blob/7a774259dcdea30ccef48eaf507f5a2fd6e91039/app/config/ezplatform.yml#L32